tests/test_requests.py

```diff
@@ -460,10 +460,20 @@ def test_HTTP_200_OK_PUT(self, httpbin):
         r = requests.put(httpbin('put'))
         assert r.status_code == 200
 
-    def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin):
-        auth = ('user', 'pass')
-        url = httpbin('basic-auth', 'user', 'pass')
+    @pytest.mark.parametrize(
+        'username, password', (
+            ('user', 'pass'),
+            (u'имя'.encode('utf-8'), u'пароль'.encode('utf-8')),
+        ))
+    def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin, username, password, auth_str):
+        auth = (username, password)
+        url = httpbin('get')
+        
+        r = Request('GET', url, auth=auth)
```

Comment:
Fixed.
I found https://github.com/kennethreitz/requests/blob/master/docs/dev/todo.rst#development-dependencies :)

And now I can see this error :(
```
py.test tests/test_requests.py
================================================= test session starts ==================================================
platform darwin -- Python 3.5.2, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: requests, inifile:
plugins: cov-2.2.1, httpbin-0.2.0, mock-0.11.0
collected 231 items

tests/test_requests.py ..............................................................FF....................................................................................X................127.0.0.1 - - [18/Nov/2016 15:23:05] "GET /stream/4 HTTP/1.1" 200 1080
X.................................................................

======================================================= FAILURES =======================================================
_____________________________ TestRequests.test_BASICAUTH_TUPLE_HTTP_200_OK_GET[user-pass] _____________________________

self = <tests.test_requests.TestRequests object at 0x104b15ef0>
httpbin = <function prepare_url.<locals>.inner at 0x104445ea0>, username = 'user', password = 'pass'

    @pytest.mark.parametrize(
        'username, password', (
            ('user', 'pass'),
            (u'имя'.encode('utf-8'), u'пароль'.encode('utf-8')),
        ))
    def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin, username, password):
        auth = (username, password)
        url = httpbin('get')

        r = requests.Request('GET', url, auth=auth)
        p = r.prepare()

        assert p.headers['Authorization'] == _basic_auth_str(username, password)

        r = requests.get(url, auth=auth)
        assert r.status_code == 200

        r = requests.get(url)
>       assert r.status_code == 401
E       assert 200 == 401
E        +  where 200 = <Response [200]>.status_code

tests/test_requests.py:481: AssertionError
------------------------------------------------- Captured stderr call -------------------------------------------------
127.0.0.1 - - [18/Nov/2016 15:23:03] "GET /get HTTP/1.1" 200 337
127.0.0.1 - - [18/Nov/2016 15:23:03] "GET /get HTTP/1.1" 200 294
 TestRequests.test_BASICAUTH_TUPLE_HTTP_200_OK_GET[\xd0\xb8\xd0\xbc\xd1\x8f-\xd0\xbf\xd0\xb0\xd1\x80\xd0\xbe\xd0\xbb\xd1\x8c]

self = <tests.test_requests.TestRequests object at 0x102240240>
httpbin = <function prepare_url.<locals>.inner at 0x103f73730>, username = b'\xd0\xb8\xd0\xbc\xd1\x8f'
password = b'\xd0\xbf\xd0\xb0\xd1\x80\xd0\xbe\xd0\xbb\xd1\x8c'

    @pytest.mark.parametrize(
        'username, password', (
            ('user', 'pass'),
            (u'имя'.encode('utf-8'), u'пароль'.encode('utf-8')),
        ))
    def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin, username, password):
        auth = (username, password)
        url = httpbin('get')

        r = requests.Request('GET', url, auth=auth)
        p = r.prepare()

        assert p.headers['Authorization'] == _basic_auth_str(username, password)

        r = requests.get(url, auth=auth)
        assert r.status_code == 200

        r = requests.get(url)
>       assert r.status_code == 401
E       assert 200 == 401
E        +  where 200 = <Response [200]>.status_code

tests/test_requests.py:481: AssertionError
------------------------------------------------- Captured stderr call -------------------------------------------------
127.0.0.1 - - [18/Nov/2016 15:23:03] "GET /get HTTP/1.1" 200 353
127.0.0.1 - - [18/Nov/2016 15:23:03] "GET /get HTTP/1.1" 200 294
=================================== 2 failed, 227 passed, 2 xpassed in 24.46 seconds ===================================
```
What is your